src/erldns_worker.erl

```diff
@@ -194,3 +194,15 @@ handle_timeout(WorkerProcessSup, WorkerProcessId) ->
             erldns_events:notify({?MODULE, restart_failed, {Error}}),
             {error, timeout}
     end.
+
+ingress_udp_request_timeout() ->
+    proplists:get_value(ingress_udp_request_timeout, env(), ?DEFAULT_UDP_PROCESS_TIMEOUT).
+
+ingress_tcp_request_timeout() ->
+    proplists:get_value(ingress_tcp_request_timeout, env(), ?DEFAULT_TCP_PROCESS_TIMEOUT).
+
+env() ->
+    case application:get_env(erldnsimple, erldns) of
```

Comment:
The environment here should be `erldns` only, not `erldnsimple`, `erldns`. See https://github.com/dnsimple/erldns/blob/main/src/erldns_config.erl - also, consider moving the configuration functions into that module.

Reply:
@aeden - https://github.com/dnsimple/erldns/pull/127/commits/313185b0b5b05f16acc40ee4fab58855b3890358 addresses this.